Header header
uint8 src
uint8 dest
float64[] mean
float64[] covariance
float64 src_ci_rate
